routes/manual.tsx

```diff
@@ -277,15 +272,21 @@ function ToC({
 export const handler: Handlers<Data> = {
   async GET(req, { params, render }) {
     const url = new URL(req.url);
+    const { version, path } = params;
+    if (!version || !path) {
+      url.pathname = `/manual@${version || versions[0]}/${
+        path || "introduction"
+      }`;
+      return Response.redirect(url);
+    }
     if (url.pathname.endsWith(".md")) {
       url.pathname = url.pathname.slice(0, -3);
       return Response.redirect(url);
     }
 
-    const version = params.version || versions[0];
     const sourceURL = getFileURL(
       version,
-      params.path ? `/${params.path}` : "/introduction",
+      `/${params.path}`,
     );
```

Comment:
nitpick:
```
    const sourceURL = getFileURL(version, `/${path}`);
```
(github doesnt allow me to comment as suggestion)

Reply:
I guess `deno fmt` is less smart than Prettier 🥲